src/client/quality_control.py

```diff
@@ -14,7 +14,7 @@
 class QualityControl(BaseTolokaObject):
     """Quality control unit settings and pool ID with training tasks
 
-    Quality control lets you get more accurate responses and restrict access to tasks for cheating performers.
+    Quality control lets you get more accurate responses and restrict access to tasks for cheating users.
```

Comment:
Why not Tolokers?

Reply:
I have seen recommendation not to use _Tolokers_ with words like _fraudulent_ and so on. And _cheating_ has similar sense. I do not insist, anyway.
@alexandervnuchkov, comment, please.

Reply:
Although, I overlooked it and changed collectors.py with _Filtering cheating Tolokers who respond too quickly_